cmd/bucket-targets.go

```diff
@@ -37,7 +37,7 @@ import (
 )
 
 const (
-	defaultHealthCheckDuration = 100 * time.Second
+	defaultHealthCheckDuration = 5 * time.Second
```

Comment:
Note: since healthcheck is being performed inside minio-go and only when network is down, should be ok to lower the duration.